x-pack/docs/en/security/backup-security-configuration.asciidoc

```diff
@@ -0,0 +1,263 @@
+* If there are indications that the snapshot has been tampered with, do not
+restore it. There is currently no option for the restore process to detect
+malicious tampering.
+
+The same recommendations apply to storing backups for configuration files.
+However, use special caution when you store the `elasticsearch.keystore` and
+private key file backups. Currently, the secure store is *not* password
+protected and any read access can compromise the passwords of external systems
+and private keys.
+
+[float]
+[[restore-security-configuration]]
+==== Restoring security configuration
+
+NOTE: You can restore a snapshot of the `.security` index only if it was created
+in the same major release version. The last minor version of every major release
+can convert and read both versions of the index.
+
+To restore your security configuration from a backup, log in to one of the node
+hosts, navigate to {es} installation directory, and follow these steps:
+
+. Make sure the repository holding the `.security` snapshot is installed.
++
+--
+[source,js]
+--------------------------------------------------
+GET /_snapshot/my_backup
+--------------------------------------------------
+// CONSOLE
+--
+
+. Add a new user with the `superuser` built-in role to the file based realm.
++
+--
+For example, create a user named `jacknich`:
+[source,shell]
+--------------------------------------------------
+bin/elasticsearch-users useradd jacknich -p password -r superuser
+--------------------------------------------------
+--
+
+. Using the previously created user, delete the existing `.security-6` or
+`.security-7` index.
++
+--
+[source,shell]
+--------------------------------------------------
+curl -u jacknich -X DELETE "localhost:9200/.security-7"
+--------------------------------------------------
+
+WARNING: After this step authentication will not work; all API calls will fail.
+
+--
+
+. Using the same user, restore the `.security` index from the snapshot.
++
+--
+[source,shell]
+--------------------------------------------------
+ curl -u jacknich -X POST "localhost:9200/_snapshot/my_backup/snapshot_1/_restore" -H 'Content-Type: application/json' -d'
+ {
+    "indices": ".security-7",
+    "include_global_state": true
+ }
+ '
+--------------------------------------------------
+
+NOTE: The `include_global_state` is optional but it will help to make sure the
+configuration in the index is compatible with the rest of the configuration in
+the cluster, as was the case at the time of the snapshot. But be advised that
+this will also restore non-security persistent cluster seetting.
+--
+
+. Optionally, cherry-pick and <<cluster-update-settings,apply the persistent
+settings>> that you have extracted with the `GET _cluster/settings` API, if you
+need to review and override the settings that were included in the snapshot (by
+the `include_global_state` flag).
+
+Lastly, if your backup included configuration files copy these and overwrite
```

Comment:
Per the above, whether this is needed depends on the nature of the incident. Do you think we can provide a bit more of a guide here?